iati_datastore/iatilib/crawler.py

```diff
@@ -393,6 +393,14 @@ def download_and_update():
     update_registry()
 
 
+@manager.cli.command('fetch-dataset-metadata')
```

Comment:
I found this initially confusing because there’s a `crawler.fetch_dataset_metadata` command that does something different.

Reply:
In fact I think this should either be called `fetch_dataset_list_cmd()`, or we should do some other renaming to make this make sense.